doc/Developing.md

```diff
@@ -62,6 +73,63 @@ It can also be started with:
 
     $ npm start
 
+## Synchronizing changes between theia and the examples
+
+The examples provided with theia are structured such that they have a file
+dependency on the theia root directory, the component in this repository is
+called theia-core.
+
+This means that the examples have their own version of theia-core in their
+node_modules directory.
+
+So each time you make a change in the theia-core you need to synchronize
+the theia-core in the example's node_modules directory.
+
+The current way to do so is to run this command in the example directory:
+
+	$ rm -rf node_modules/theia && npm install
```

Comment:
FYI: executing the `npm run watch` command on `theia-core` and then starting the browser example with `npm run start` includes the hot reload functionality. So if you change the client code in `theia-core` your browser get refreshed with the most recent content. It is unfortunate that it does not work correctly for you.